### DISCUSSION TITLE
GitHub Education

### DISCUSSION THREAD

**Discussion by sLizzard1234** [community/community#58733] [GitHub Education]
### Hi everyone,

Hello,

I'm experiencing difficulties in applying for student benefits. I have included a brief bio and my legal name, and I have also attached a photo of my student ID. However, I keep receiving a response email stating: 

**Hey sLizzard,

Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Jun 22, 2023 for [@sLizzard1234](https://github.com/sLizzard1234).

    You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.

Once you have resolved the issues with your sumbission, [visit us again and reapply](https://education.github.com/).

Have questions? Join the [GitHub Education Community](https://github.com/orgs/community/discussions/categories/github-education) for answers.

💖 The GitHub Education Team**

Is anyone else facing the same issue and, if so, does anyone have a potential solution?

**Comment by samus-aran** [comment]
Hi @sLizzard1234 👋 🦎 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

Hope this helps 👍
